### ISSUE TITLE
[iceprod2] password protect admin web pages (Trac #1500)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1492]
As a basic step before true user/group support in #841, password protect all the admin-like web pages in iceprod.  basic authentication should work fine.

Migrated from https://code.icecube.wisc.edu/ticket/1500
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "As a basic step before true user/group support in #847, password protect all the admin-like web pages in iceprod.  basic authentication should work fine.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] password protect admin web pages", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-01-06T21:12:03", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160107T22:30:13: 
* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 140755:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140755"
> add an admin password to the website.  closes #1492. this also provides the frontend login structure for #841.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
